dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_logs_view.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useCallback, useContext, useMemo } from 'react';
+import { EuiAccordion, EuiSpacer } from '@elastic/eui';
+import { IConfigPanelOptionSection } from '../../../../../../../../common/types/explorer';
+import { ButtonGroupItem } from './config_button_group';
+import { TabContext } from '../../../../../../event_analytics/hooks';
+
+export const ConfigLogsView = ({
+  visualizations,
+  schemas,
+  vizState,
+  handleConfigChange,
+  sectionName,
+  sectionId = 'chartStyles',
+}) => {
+  const { explorerData } = useContext<any>(TabContext);
+  const rawData = explorerData.jsonData;
+
+  const handleConfigurationChange = useCallback(
+    (stateFiledName) => {
+      return (changes) => {
+        handleConfigChange({
+          ...vizState,
+          [stateFiledName]: changes,
+        });
+      };
+    },
+    [handleConfigChange, vizState]
+  );
+
+  const dimensions = useMemo(
+    () =>
+      schemas.map((schema: IConfigPanelOptionSection, index: string) => {
+        let params = {
+          title: schema.name,
+          vizState,
+          ...schema.props,
+        };
+        const DimensionComponent = schema.component || ButtonGroupItem;
+        if (schema.eleType === 'buttons') {
+          params = {
+            title: schema.name,
+            legend: schema.name,
+            groupOptions: schema?.props?.options.map((btn: { name: string }) => ({
+              ...btn,
+              label: btn.name,
+            })),
+            idSelected: vizState[schema.mapTo] || schema?.props?.defaultSelections[0]?.id,
+            handleButtonChange: handleConfigurationChange(schema.mapTo),
+            vizState,
+            ...schema.props,
```

Comment:
Usually we do the destructuring at the beginning of the object, so that in case some of the properties exist in the de-structured object, they get overwritten by our values and the ones we define specifically here, exist in the final object. I hope, you understand what you are doing and there are no properties with the same name inside schema.props.